### PR TITLE
Liquid Edit Filters

### PR DESCRIPTION
**Pull request by thescubageek**
Adds the following edit filters for MUCH easier management of edit.html for widgets:

* edit_text_field
* edit_url_field
* edit_color_field
* edit_integer_field
* edit_textarea_field
* edit_radio_field
* edit_toggle_field
* edit_row_layout_widget_field
* edit_col_layout_widget_field
* edit_select_field
* edit_page_field
* edit_location_field
* edit_photo_fields
* edit_gallery_fields
* edit_save_button
* edit_goback_button

The following Widgets have been updated to use the new Edit filters:

* Button
* Content Stripe
* Column
* Mini News Feed
* Gallery

### CODE REVIEW

**Comment by thescubageek** [comment]
@dreuss @johnlucia @jcstringer @levibrown @mmitchellg5 

New liquid filters for Widget Edit fields...... cleans up a LOT of duplicate code in widgets and consolidates field markup into these filters.

I've ported the 5 widgets that I feel have a good broad representation of what we do with our edit forms for widgets.

File: app/models/website.rb [hunk 1]
```diff
@@ -101,4 +101,8 @@ def single_domain_location?
   def client
     @client ||= Client.first
   end
+
+  def web_pages
```
Comment (variousred) [inline app/models/website.rb]: i believe #web_templates will get you the same thing

**Comment by variousred** [comment]
awesome refactor!

File: app/models/website.rb (hunk 1 above)
Comment (thescubageek) [inline app/models/website.rb]: web_templates also returns the website_template object for some reason.... had to use just the home + pages to get this to have the correct data

**Comment by variousred** [comment]
:shipit:

**Pull request merged**